resources/js/index.tsx

```diff
@@ -1,48 +1,57 @@
 /* @refresh reload */
-import { render } from "solid-js/web";
+import { TransProvider } from "@mbarzda/solid-i18next";
+import { MetaProvider } from "@solidjs/meta";
 import { Router } from "@solidjs/router";
+import { InitializeTanstackQuery } from "components/utils";
 import i18next from "i18next";
-import { TransProvider } from "@mbarzda/solid-i18next";
 import I18NextHttpBackend from "i18next-http-backend";
+import { Show, createEffect, createSignal } from "solid-js";
+import { render } from "solid-js/web";
 import { Toaster } from "solid-toast";
-
-import "./index.scss";
 import App from "./App";
-import { MetaProvider } from "@solidjs/meta";
+import "./index.scss";
 
 const root = document.getElementById("root");
 
 if (!(root instanceof HTMLElement)) {
-    throw new Error(
-        "Root element not found. Did you forget to add it to your index.html? Or maybe the id attribute got mispelled?"
-    );
+  throw new Error(
+    "Root element not found. Did you forget to add it to your index.html? Or maybe the id attribute got mispelled?"
+  );
 }
 
 render(() => {
-    i18next.use(I18NextHttpBackend);
+  const [transLoaded, setTransLoaded] = createSignal(false);
+  i18next.use(I18NextHttpBackend);
+
+  createEffect(() => i18next.on("loaded", () => setTransLoaded(true)));
```

Comment:
Dlaczego to jest w effekcie? Nie można tego napisać normalnie? Chyba setter zadziała nawet jak się go woła poza reaktywnym kontekstem?

Reply:
trochę się pospieszyłem. Metoda 'on' zbytnio skojarzyła mi się z klasycznym event listenerem i pomyślałem, że muszę anulować listener w createEffect 🤕 
Jest to jak najbardziej zbędne